context/app/static/js/components/Search/__tests__/accordionFilters.spec.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
Should the list filter/range filter cases be tested?

Reply:
yes they should. More tech debt. Sorry.